terraform/environments/production/main.tf

```
@@ -786,7 +786,7 @@ module "relays" {
   image      = "relay"
   image_tag  = var.image_tag
 
-  observability_log_level = "debug,firezone_relay=trace,hyper=off,h2=warn,tower=warn,wire=trace"
+  observability_log_level = "debug,hyper=off,h2=warn,tower=warn"
```

@conectado @thomaseizinger isn't `wire` is still very noisy with debug? WDYT about setting level to `info` on prod and only setting levers lower than that for specific libraries?

```suggestion
  observability_log_level = "debug,hyper=off,h2=warn,tower=warn,wire=warn"
```

---

No, `wire` is only active on `trace` level, at least within the relay.

---

> WDYT about setting level to `info` on prod and only setting levers lower than that for specific libraries?

I think that is a good long-term plan. I am curious to see what the performance is like with `debug`. At least on the relay, we will print one line per UDP packet on `debug`. Perhaps that is too much? I guess only benchmarking will tell.

---

Don't forget that we also pay for the log storage, and one per packet is a lot of IO pressure to write those

---

Yeah true. I'll think of something better!

---

I ended up implementing a timer like @conectado had/has it in the tunnel as well: https://github.com/firezone/firezone/pull/3408. With this, I think we can comfortably set this to `info` and we will get _some_ insight about what is happening without spamming the log storage too much.